src/utils/azureClients.ts

```diff
@@ -8,6 +8,7 @@ import { WebSiteManagementClient } from '@azure/arm-appservice';
 import { CosmosDBManagementClient } from '@azure/arm-cosmosdb';
 import { EventHubManagementClient } from '@azure/arm-eventhub';
 import { ServiceBusManagementClient } from '@azure/arm-servicebus';
+import { SqlManagementClient } from '@azure/arm-sql';
```

Comment:
All of these `@azure` client imports should be `import type`, otherwise there's no gain by doing `await import` in the code below.